classes/class-3-4/matias.css

```diff
@@ -0,0 +1,238 @@
+    background: #0F2027;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #2C5364, #203A43, #0F2027);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #2C5364, #203A43, #0F2027); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+body::-webkit-scrollbar{
+    display: none;
+}
+/* *************************************************************************************************
+*************************************************************************************************
+************************************************************************************************* */
+
+header{
+    width: 100%;
+    height: 80px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 1.5rem;
+    text-align: center;
+    color: #fff;
+}
+
+/* *************************************************************************************************
+*************************************************************************************************
+************************************************************************************************* */
+
+nav{
+    width: 100%;
+    height: 50px;
+    display: flex;
+    justify-content: space-evenly;
+    border: 1px solid #fff;
+    /* border-radius: 2rem; */
+    margin: 1rem auto;
+    background: #C6FFDD;
+}
+
+ul{
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    list-style: none;
+    font-size: 1.5rem;
+    color: #203A43;
+    font-weight: bold;
+}
+
+li{
+    cursor: pointer;
+    position: relative;
+    /* transition: all .3s; */
+    display: inline-block;
+}
+
+/* *************************************************************************************************
+*************************************************************************************************
+************************************************************************************************* */
+
+main{
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr;
+    grid-gap: 2rem;
+    height: 100%;
+    padding: 2rem;
+    align-items: center;
+    justify-items: stretch;
+    overflow: auto;
+}
+
+section{
+    border-radius: 2rem;
+    min-height: 300px;
+    background-color: #3d3d3d;
+    padding: 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    overflow-x: auto;
+}
+section::-webkit-scrollbar{
+    display: none;
+}
+
+section h2{
+    text-align: center;
+    color: #fff;
+    font-size: 1.5rem;
+    text-decoration: underline wavy;
+}
+
+.section-content{
+    display: flex;
+    flex-wrap: nowrap;
+    min-height: 300px;
+    width: 900px;
+    gap: 1rem;
+    overflow: visible;
+    padding: 1rem;
+    scroll-snap-type: x mandatory;
+    scroll-padding: 50%;
+}
+
+
+.section-content::-webkit-scrollbar{
+    display: hide;
+}
+
+article{
+    min-height: 100%;
+    flex: 1 1 300px;
+    border-radius: 1rem;
+    scroll-snap-align: center;
+}
+
+.nro1{
```

Comment:
Is a good practice to avoid numbers on ids and classes, you could replace it with 'first', 'second'...